lib/containers/podman.pm

```diff
@@ -27,7 +27,8 @@ sub configure_insecure_registries {
 
     assert_script_run "curl " . data_url('containers/registries.conf') . " -o /etc/containers/registries.conf";
     assert_script_run "chmod 644 /etc/containers/registries.conf";
-    file_content_replace("/etc/containers/registries.conf", REGISTRY => $registry);
+    # Add custom registry only if it is not docker.io, which is already present in registries.conf and docker.io is the default return value for get_var('REGISTRY')
```

Comment:
```suggestion
    # Add custom registry only if it is not docker.io, which is already present in registries.conf and docker.io is the default return value for registry_url();
```

Reply:
The whole comment got replaced in the meantime, following your previous comment